app/controllers/students_controller.rb

```diff
@@ -0,0 +1,51 @@
+class StudentsController < ApplicationController
+	before_action :authenticate_teacher!,only: [:edit,:update,:destory,:list]
+	before_action :authenticate_student!,only: [:welcome]
+	before_action :set_student, only: [:edit, :update, :destroy,:show]
+	def edit
+	end
+	def show
+		
+	end
+	def new
+		@student=Student.new
+	end
+	def create
+    	@student=Student.new(student_params)
+
+    	if @student.save
+    		 no=Time.now.year.to_s+""+@student.standard_id.to_s+""+@student.id.to_s
+    		 puts "noooo--------->"+no.to_s
```

Comment:
What is no?
Parameter name should show it's purpose.

Set appropriate name as per value assign into it.

remove puts.